src/server/frontend_xwayland/xcb_connection.cpp

```diff
@@ -231,18 +230,23 @@ auto mf::XCBConnection::read_property(
         0, // no offset
         2048); // big buffer
 
-    return [this, cookie, action, on_error, window, prop]()
+    return [this, cookie, handler=std::move(handler), window, prop]()
         {
             try
             {
+                ErrorPtr error;
```

Comment:
Looks like this is default initialized and not written before it's read. Can that be right?

Reply:
Oops, actual usage got dropped in a rebase somewhere along the line.